packages/cli/src/models/files/ProjectFile.ts

```diff
@@ -22,53 +24,68 @@ interface ConfigFileCompilerOptions {
   };
 }
 
-export default class ProjectFile {
-  public fileName: string;
-  public data: {
-    name: string;
-    version: string;
-    manifestVersion: string;
-    dependencies: { [name: string]: string };
-    contracts: { [alias: string]: string };
-    publish: boolean;
-    compiler: ConfigFileCompilerOptions;
-  };
+interface ProjectFileData {
+  name: string;
+  version: string;
+  manifestVersion: string;
+  dependencies: { [name: string]: string };
+  contracts: { [alias: string]: string };
+  publish: boolean;
+  compiler: ConfigFileCompilerOptions;
+}
+
+export const PROJECT_FILE_NAME = 'project.json';
+export const PROJECT_FILE_PATH = path.join(
+  OPEN_ZEPPELIN_FOLDER,
+  PROJECT_FILE_NAME,
+);
+export const LEGACY_PROJECT_FILE_NAME = 'zos.json';
 
-  public static getLinkedDependencies(fileName: string = 'zos.json'): string[] {
-    const file = fs.parseJsonIfExists(fileName);
-    if (file && file.dependencies) {
-      return Object.keys(file.dependencies).map(
-        depName => `${depName}@${file.dependencies[depName]}`,
+export default class ProjectFile {
+  public filePath: string;
+  public data: ProjectFileData;
+
+  public static getLinkedDependencies(filePath: string = null): string[] {
+    const project = new ProjectFile(filePath);
+    if (project && project.dependencies) {
+      return Object.keys(project.dependencies).map(
+        depName => `${depName}@${project.dependencies[depName]}`,
       );
-    } else return [];
-  }
-
-  public constructor(fileName: string = 'zos.json') {
-    this.fileName = fileName;
-    try {
-      this.data = fs.parseJsonIfExists(this.fileName) || {
-        manifestVersion: MANIFEST_VERSION,
-      };
-      // if we failed to read and parse zos.json
-    } catch (e) {
-      e.message = `Failed to parse '${path.resolve(
-        fileName,
-      )}' file. Please make sure that ${fileName} is a valid JSON file. Details: ${
-        e.message
-      }.`;
-      throw e;
     }
-    checkVersion(this.data.manifestVersion, this.fileName);
+    return [];
+  }
+
+  public constructor(filePath: string = null) {
```

Comment:
```suggestion
  constructor(filePath: string = null) {
```

Reply:
Having explicit constructor is a [typescript-eslint](https://github.com/typescript-eslint/typescript-eslint/blob/v1.9.0/packages/eslint-plugin/docs/rules/no-explicit-any.md).